src/test/components/BeforeUnloadManager.test.js

```diff
@@ -4,10 +4,12 @@
 
 // @flow
 import * as React from 'react';
-import { render, fireEvent } from '@testing-library/react';
+import { fireEvent } from '@testing-library/react';
```

Comment:
Actually you can import `fireEvent` from our wrapper as well. Indeed we reexport all exports from testing-library with the line `export * from '@testing-library/react'`. Same for `screen` below.
Do you think you can have a look at that?

Reply:
Actually don't look at it, it's not important, we can change that when we'll get to it.

Reply:
Should I change all the imports for `fireEvent` and `screen` from `fixtures/testing-library`? I can do that in a new PR.

Reply:
don't worry about it :-)

Initially I thought we could just search and replace `'@testing-library/react'` to `'firefox-profiler/test/fixtures/testing-library'` in all the code, but now that you went through the work of changing all of them manually, let's not do that again, this is tedious and doesn't bring much.

Reply:
oh, I seeee. I could have just change the import location even for the `render` with other functions too 🤦‍♀️